swift_browser_ui_frontend/src/components/ShareModal.vue

```diff
@@ -423,15 +431,20 @@ export default {
   }
 
   c-select {
-    color: var(--csc-dark-grey);
+    color: $csc-dark-grey;
   }
 
   c-link > span {
     font-size: 0.875rem;
   }
 
-  .field {
-    margin: 2rem 0 0 0;
+  #shareid-tags {
+    margin: 1rem 0;
+    border: 1px solid $csc-dark-grey;
+  }
+
+  #shareid-tags:focus-within {
+    border: 2px solid $csc-primary;
```

Comment:
Is this one also needed if the component already has the same styling?

Reply:
This relates to my other comment. Since I already put the border for Taginput in this ShareModal, I need to specify the other border when it is focused.